exec/key-handler

```diff
@@ -43,6 +43,13 @@ tag_del() {
 	rm -f "$TMPFILE"
 }
 
+help() {
+  ( printf 'Keybindings inside C-x prefix:\n\n';
+    sed -n -e '/^case /,/^esac/{//!p}' "$0" # assumes GNU sed
```

Comment:
Alternatively, `sed -n -e '/^case /,/^esac/{/^case /d;/^esac/d;p;}' "$0"` works with BSD sed but duplicates the pattern.

Thanks for writing sxiv, it's a pleasure to have it in my toolbox.